facts/inventory.py

```diff
@@ -509,9 +551,32 @@ def generatekeaconfig(servers, aps, vlans, outputdir):
 
     kea_config["Dhcp4"]["subnet4"] = subnets_dict
     kea_config["Dhcp4"]["reservations"] = reservations_dict
-    with open(f'{outputdir}/kea.json', 'w') as f:
+
+    with open(f'{outputdir}/dhcp4-server.conf', 'w') as f:
         f.write(json.dumps(kea_config, indent=2))
 
+    subnets6_dict = [
+        {
+            "subnet": vlan["ipv6prefix"] + "/" + str(vlan["ipv6bitmask"]),
+            # generating uniq id (prefix with dots) for each subnet block to ensure autoids dont effect reordering
+            # called out in: https://kea.readthedocs.io/en/latest/arm/dhcp4-srv.html#ipv4-subnet-identifier
+            # subnet ids must be greater than zero and less than 4294967295
+            #
+            # for ipv6 well take the last 8 hex digits to make sure the id is small enough but uniq
```

Comment:
Given that we are starting from a /48 (and utterly unlikely to ever go beyond that) and issuing /64 VLANs, the last 4 hex digits is sufficient to be guaranteed unique.

Obviously, we need to special case the ...:0::/64 prefix if we ever use that for something DHCP, but otherwise, we should be able to use the straight subnet number.

Reply:
thats sounds good, well go with 4 hex digits

Reply:
This was updated to 4